docs/src/AvatarGroup.doc.js

```diff
@@ -0,0 +1,506 @@
+            },
+          ],
+          [
+            {
+              name: 'Keerthi',
+              src: 'https://i.ibb.co/ZfCZrY8/keerthi.jpg',
+            },
+            {
+              name: 'Alberto',
+              src: 'https://i.ibb.co/NsK2w5y/Alberto.jpg',
+            },
+          ],
+          [
+            {
+              name: 'Keerthi',
+              src: 'https://i.ibb.co/ZfCZrY8/keerthi.jpg',
+            },
+            {
+              name: 'Alberto',
+              src: 'https://i.ibb.co/NsK2w5y/Alberto.jpg',
+            },
+            {
+              name: 'Shanice',
+              src: 'https://i.ibb.co/7tGKGvb/shanice.jpg',
+            },
+          ],
+          [
+            {
+              name: 'Keerthi',
+              src: 'https://i.ibb.co/ZfCZrY8/keerthi.jpg',
+            },
+            {
+              name: 'Alberto',
+              src: 'https://i.ibb.co/NsK2w5y/Alberto.jpg',
+            },
+            {
+              name: 'Shanice',
+              src: 'https://i.ibb.co/7tGKGvb/shanice.jpg',
+            },
+            ...new Array(10),
+          ],
+          [
+            {
+              name: 'Keerthi',
+              src: 'https://i.ibb.co/ZfCZrY8/keerthi.jpg',
+            },
+            {
+              name: 'Alberto',
+              src: 'https://i.ibb.co/NsK2w5y/Alberto.jpg',
+            },
+            {
+              name: 'Shanice',
+              src: 'https://i.ibb.co/7tGKGvb/shanice.jpg',
+            },
+            ...new Array(100),
+          ],
+        ]}
+      >
+        {({ addCollaborators, collaborators }) => {
+          const accessibilityLabel =
+            collaborators.length <= 3
+              ? `Collaborators: ${collaborators.map((x) => x?.name).join(', and ')}.`
+              : `Collaborators: ${collaborators
+                  .slice(0, 2)
+                  .map((x) => x?.name)
+                  .join(', ')} ${
+                  collaborators.length > 3 ? `and ${collaborators.length - 2} more.` : '.'
+                }`;
+          return addCollaborators ? (
+            <AvatarGroup
+              accessibilityLabel={`${accessibilityLabel} Add collaborators to this board.`}
+              size="md"
+              collaborators={collaborators}
+              addCollaborators
+              onClick={() => {}}
+              role="button"
+            />
+          ) : (
+            <AvatarGroup
+              accessibilityLabel={accessibilityLabel}
+              size="md"
+              collaborators={collaborators}
+            />
+          );
+        }}
+      </CombinationNew>
+    </MainSection.Subsection>
+    <MainSection.Subsection
+      columns={2}
+      title="Role"
+      description="AvatarGroup can be display only, but can also act as a button or link. It will only be clickable if role is set to `button` or `link`. For button role, `onClick is required. For link role, `href` is required."
```

Comment:
Missing backtick after "onClick" is causing funkiness:
![Screen Shot 2021-04-29 at 11 04 38 AM](https://user-images.githubusercontent.com/12059539/116627247-997b7b00-a901-11eb-9372-34998dfa5320.png)

Reply:
DONE